client/jetpack-connect/style.scss

```diff
@@ -777,3 +776,167 @@ $colophon-height: 50px;
 .jetpack-connect__connect-button {
 	text-align: center;
 }
+
+/**
+ * Onboarding styles
+ **/
+.is-section-jetpack-connect:not(.is-add-site-page),
+.is-jetpack-login {
+	background-color: var( --color-jetpack-onboarding-background );
+
+	&.layout {
+		position: relative;
+		min-height: calc( 100% - #{$colophon-height} );
+		padding-bottom: $colophon-height;
+	}
+
+	.login__form-header,
+	.formatted-header {
+		color: var( --color-jetpack-onboarding-text );
+	}
+
+	.jetpack-logo__text,
+	.wpcom-colophon path {
```

Comment:
Should we also Jetpackify:

Thank you page progress bar:
![](https://cldup.com/Gc1MXgeYC1.png)

Inline help button:
![](https://cldup.com/eelHtBUfcB.png)

Inline help popover:
![](https://cldup.com/UWcl0UFC-N.png)

Reply:
What do you mean by “Jetpackify”?

Reply:
😉 Sorry for the confusion. I meant replacing the default subtle blue colors with subtle green colors from the `--color-jetpack*` family.

Reply:
I’d avoid customizing the help button and popover.

Reply:
Following Shaun's guidance here (some reasons why would be much appreciated still), so only updated the progress bar indicator in https://github.com/Automattic/wp-calypso/pull/31141/commits/fe006ca3464b95db28c21c9b49c0f3fc20700092

Reply:
Sounds good to me, as long as it's been thought of 👍